network/tests/common.py

```diff
@@ -18,8 +18,10 @@
     'system.net.bytes_rcvd',
     'system.net.bytes_sent',
     'system.net.packets_in.count',
+    'system.net.packets_in.drop',
     'system.net.packets_in.error',
     'system.net.packets_out.count',
+    'system.net.packets_out.drop',
     'system.net.packets_out.error',
 ]
```

Comment:
Same as in my previous comment, could you fix this so that this would only contain the new metrics on platforms where we expect them?

Reply:
Fixed at https://github.com/DataDog/integrations-core/pull/10551/commits/55a985cd7f4ba3aa4b5f7332d0677d3c75b9859d#diff-b730719180db0fbb6ea35b61aaa8857ba145a9806cd97bfc42f9450c95d74d4eR27-R32